lizard_auth_client/models.py

```diff
@@ -82,6 +83,14 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @property
+    def uuid(self):
+        return UUID(self.unique_id)
```

Comment:
Ik vroeg me even af of dit steeds een nieuwe uuid ging genereren, maar dat is niet zo. Dus da's goed.